src/lua/CommunityBalanceMod/FortressPvE/TwiliteShade/TechData.lua

```diff
@@ -0,0 +1,64 @@
+
+local oldBuildTechData = BuildTechData
+function BuildTechData()
```

Comment:
Open question: Is it better to keep this separate from the combined TechData.lua or should we combine them?

Reply:
Twilites code will be merged once we know after a playtest that there are no changes needed and final. 
For now lets keep it like this to be able to react to updates.